src/esp/sensor/Sensor.cpp

```diff
@@ -35,10 +34,10 @@ void SensorSpec::sanityCheck() const {
                      sensorSubType < SensorSubType::SensorSubTypeCount,
                  "SensorSpec::sanityCheck(): sensorSubType"
                      << int32_t(sensorType) << "is illegal", );
-  CORRADE_ASSERT((abs(position.array()) >= 0).any(),
-                 "SensorSpec::sanityCheck(): position is illegal", );
-  CORRADE_ASSERT((abs(orientation.array()) >= 0).any(),
-                 "SensorSpec::sanityCheck(): orientation is illegal", );
+  // CORRADE_ASSERT((abs(position) >= 0).any(),
+  //                "SensorSpec::sanityCheck(): position is illegal", );
```

Comment:
`(Math::abs(position) >= Vector3{0.0f}).any()` and similarly below should work, but I'm not sure what the check is supposed to actually do. So maybe a better message also? :D